code/modules/client/client procs.dm

```diff
@@ -392,6 +395,14 @@
 		if(ckey != query_cid.item[1])
 			related_accounts_cid.Add("[query_cid.item[1]]")
 
+	var/admin_rank = "Player"
+	if(holder)
+		admin_rank = holder.rank
+	// Admins don't get slammed by this, I guess
+	else
+		if(check_randomizer())
```

Comment:
The hit from the detector *does* get logged, but their player info isn't updated in the database (what this proc, `log_client_to_db`, does), because their client is deleted before that occurs.